boards/x86/qemu_x86/Kconfig.defconfig

```diff
@@ -53,10 +49,6 @@ config BOARD
 config KERNEL_VM_SIZE
 	default 0x10000000 if ACPI
 
-config HEAP_MEM_POOL_SIZE
-	default 1048576 if ACPI
-	depends on KERNEL_MEM_POOL
-
```

Comment:
So qemu does not use ACPI anymore?

Reply:
> So qemu does not use ACPI anymore?

No it does. The Kconfig option for ACPI comes from `lib/acpi/Kconfig` and since its default value is 1M (which is what qemu used) we don't need to modify this default in the qemu defconfig anymore.